### DISCUSSION TITLE
How to bind a command to the keyboard key "Backspace"

### DISCUSSION THREAD

**Discussion by gaberiel44** [d2phap/ImageGlass#1610] [Q&A] [answered]
I tried:
```
    "MnuGoToFirst": [
      "Backspace"
    ],
```
but it gets turned into:
```
    "MnuGoToFirst": [
      "None"
    ],
````
I tried looking for the correct keyword in the code base here, I did not find it.

Also is there a place where I can get a complete list of keyboard keys and mouse keys as ImageGlass expects them in the Json file? 

Thank you

**Comment by d2phap** [comment, marked as answer]
Can you try “Back” instead of “Backspace”

**Reply by gaberiel44** [reply]
Perfect, that does it. Cheers!
